satellite/audit/worker.go

```diff
@@ -34,13 +34,14 @@ func NewWorker(log *zap.Logger, queue *Queue, config Config) (*Worker, error) {
 // Run runs audit service 2.0.
 func (worker *Worker) Run(ctx context.Context) (err error) {
 	defer mon.Task()(&ctx)(&err)
-	worker.log.Info("starting")
+	worker.log.Debug("starting")
 
-	// wait for all audits to run
+	// Wait for all audits to run.
```

Comment:
Just to let you know, I haven't commented on these ones because they are purely source code comments.

The ones that we ask for ends with a period are the ones which are used by godoc to generate the HTML documentation and for following the Go conventions.

For the internal ones, at least for, in my opinion, we can be more flexible. Obviously adding a period doesn't hurt.